[2025-03-10T08:02:07.819+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2025-03-10T08:02:08.054+0000] {taskinstance.py:2614} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: validate_all_tables.customers manual__2025-03-10T08:01:45.227024+00:00 [queued]>
[2025-03-10T08:02:08.188+0000] {taskinstance.py:2614} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: validate_all_tables.customers manual__2025-03-10T08:01:45.227024+00:00 [queued]>
[2025-03-10T08:02:08.190+0000] {taskinstance.py:2867} INFO - Starting attempt 1 of 2
[2025-03-10T08:02:11.756+0000] {taskinstance.py:2890} INFO - Executing <Task(PythonOperator): customers> on 2025-03-10 08:01:45.227024+00:00
[2025-03-10T08:02:11.814+0000] {warnings.py:112} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70: DeprecationWarning: This process (pid=4010) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2025-03-10T08:02:11.830+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'validate_all_tables', 'customers', 'manual__2025-03-10T08:01:45.227024+00:00', '--job-id', '115', '--raw', '--subdir', 'DAGS_FOLDER/validate_all_tables_dag.py', '--cfg-path', '/tmp/tmp50n3ky5i']
[2025-03-10T08:02:11.848+0000] {standard_task_runner.py:105} INFO - Job 115: Subtask customers
[2025-03-10T08:02:11.844+0000] {standard_task_runner.py:72} INFO - Started process 4025 to run task
[2025-03-10T08:02:12.654+0000] {task_command.py:467} INFO - Running <TaskInstance: validate_all_tables.customers manual__2025-03-10T08:01:45.227024+00:00 [running]> on host efacfdd0be48
[2025-03-10T08:02:15.167+0000] {taskinstance.py:3134} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='validate_all_tables' AIRFLOW_CTX_TASK_ID='customers' AIRFLOW_CTX_EXECUTION_DATE='2025-03-10T08:01:45.227024+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2025-03-10T08:01:45.227024+00:00'
[2025-03-10T08:02:15.187+0000] {logging_mixin.py:190} INFO - Task instance is in running state
[2025-03-10T08:02:15.199+0000] {logging_mixin.py:190} INFO -  Previous state of the Task instance: queued
[2025-03-10T08:02:15.216+0000] {logging_mixin.py:190} INFO - Current task name:customers state:running start_date:2025-03-10 08:02:08.066688+00:00
[2025-03-10T08:02:15.229+0000] {logging_mixin.py:190} INFO - Dag name:validate_all_tables and current dag run status:running
[2025-03-10T08:02:15.239+0000] {taskinstance.py:732} INFO - ::endgroup::
[2025-03-10T08:02:16.533+0000] {logging_mixin.py:190} INFO - Customers validation and cleaning successful
[2025-03-10T08:02:16.649+0000] {python.py:240} INFO - Done. Returned value was: None
[2025-03-10T08:02:16.748+0000] {taskinstance.py:341} INFO - ::group::Post task execution logs
[2025-03-10T08:02:16.761+0000] {taskinstance.py:353} INFO - Marking task as SUCCESS. dag_id=validate_all_tables, task_id=customers, run_id=manual__2025-03-10T08:01:45.227024+00:00, execution_date=20250310T080145, start_date=20250310T080208, end_date=20250310T080216
[2025-03-10T08:02:16.950+0000] {logging_mixin.py:190} INFO - Task instance in success state
[2025-03-10T08:02:16.958+0000] {logging_mixin.py:190} INFO -  Previous state of the Task instance: running
[2025-03-10T08:02:16.960+0000] {logging_mixin.py:190} INFO - Dag name:validate_all_tables queued_at:2025-03-10 08:01:45.258507+00:00
[2025-03-10T08:02:16.972+0000] {logging_mixin.py:190} INFO - Task hostname:efacfdd0be48 operator:PythonOperator
[2025-03-10T08:02:17.155+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2025-03-10T08:02:17.218+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
